app/src/main/java/org/systers/mentorship/remote/services/TaskService.kt

```diff
@@ -12,8 +14,11 @@ interface TaskService {
     /**
      * This function gets all the tasks from a mentorship relation
      * @param relationId id of the mentorship relation in question
-     * @return an observable instance of a list of [Task]s
+     * @return an Observable of a list of [Task]
      */
     @GET("mentorship_relation/{relation_id}/tasks")
     fun getAllTasksFromMentorshipRelation(@Path("relation_id") relationId: Int): Observable<List<Task>>
+
+    @PUT("mentorship_relation/{request_id}/task/{task_id}/complete")
+    fun completeTask(@Path("request_id") requestId: Int, @Path("task_id") taskId: Int): Observable<CustomResponse>
```

Comment:
Add documentation above this method, like other documentations present in this file